perun/templates/diff_sections/section_traces_table.html.jinja2

```diff
@@ -1,22 +1,26 @@
 {% import 'macros/section_title.html.jinja2' as m_title %}
 
-{% set traces_table_info = "Click on a row to explore top N calling contexts (traces) of the selected function.
+{% set traces_table_info = "Mouseover a column header to display a tooltip explaining the column.
+            Click on a row to explore top N calling contexts (traces) of the selected function.
             Click on (+) next to a trace row to open a detail about that concrete trace.
             You may filter the table using the select form in the table header." %}
 
 <div id="traces" class="section">
 
     {{ m_title.display_title("Traces Table", "traces table", traces_table_info, notes_enabled) }}
 
-    <table id="table" class="traces_table" style="width: 100%;">
+    <table id="table" class="traces_table display" style="width: 100%;">
         <thead>
             <tr>
                 <th></th>
                 <th>Unit</th>
                 <th>Change</th>
                 <th>Metric</th>
-                <th>Absolute Δ</th>
-                <th>Relative Δ</th>
+                <th title="The amount of resources consumed by the Unit in the baseline profile.">Baseline</th>
```

Comment:
Looking at this get me thinking: should we extract this (I mean the columns) to some constant/variable in the code? That could be more maintenable if we decide to do this differently.